test/fixtures/3.0/ping/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "ping",
+  "version": "1.0.0",
+  "description": "ping",
+  "keywords": [
+    "loopback-application",
+    "loopback"
+  ],
+  "main": "index.js",
+  "engines": {
+    "node": ">=8.9"
+  },
+  "scripts": {
+    "build:apidocs": "lb-apidocs",
+    "build": "lb-tsc es2017 --outDir dist",
+    "build:watch": "lb-tsc --watch",
+    "clean": "lb-clean dist",
+    "lint": "npm run prettier:check && npm run tslint",
+    "lint:fix": "npm run tslint:fix && npm run prettier:fix",
+    "prettier:cli": "lb-prettier \"**/*.ts\" \"**/*.js\"",
+    "prettier:check": "npm run prettier:cli -- -l",
+    "prettier:fix": "npm run prettier:cli -- --write",
+    "tslint": "lb-tslint",
+    "tslint:fix": "npm run tslint -- --fix",
+    "pretest": "npm run clean && npm run build",
+    "test": "lb-mocha --allow-console-logs \"dist/__tests__\"",
+    "posttest": "npm run lint",
+    "test:dev": "lb-mocha --allow-console-logs dist/__tests__/**/*.js && npm run posttest",
+    "migrate": "node ./dist/migrate",
+    "prestart": "npm run build",
+    "start": "node .",
+    "prepublishOnly": "npm run test"
+  },
+  "repository": {
+    "type": "git"
+  },
+  "author": "",
+  "license": "",
+  "files": [
+    "README.md",
+    "index.js",
+    "index.d.ts",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "dependencies": {
+    "@loopback/boot": "^1.0.12",
+    "@loopback/context": "^1.5.1",
+    "@loopback/core": "^1.1.6",
+    "@loopback/openapi-v3": "^1.2.1",
+    "@loopback/repository": "^1.1.5",
+    "@loopback/rest": "^1.5.5",
+    "@loopback/rest-explorer": "^1.1.8",
+    "@loopback/service-proxy": "^1.0.8"
+  },
+  "devDependencies": {
```

Comment:
It is not clear which part of the new codebase is responsible for installing these dependencies?

IMO, we should move all relevant dependencies to the top-level `package.json` file and remove this `package.json` file completely.